templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -2,6 +2,9 @@ mode: 0644
 path: "/etc/crio/crio.conf.d/00-default"
 contents:
   inline: |
+    [crio]
+    internal_wipe = true
```

Comment:
Can you motivate the relationship of this change to the bug a bit more?  Is it that in order to have the policy of rejecting containers that mount `/etc/hostname` apply after an upgrade we need to clear the containers after reboot?

(What's the relationship of this and the `crio-wipe.service`?  Are both needed?)

Reply:
oh shoot this needs a rebase, this is unrelated but was built off of it to prevent conflicts